test/src/test/java/org/corfudb/integration/CorfuReplicationMultiSourceSinkIT.java

```diff
@@ -302,9 +295,9 @@ protected void verifySnapshotAndLogEntrySink(boolean changeRole) throws Exceptio
         for (int i = 0; i < numSinkClusters; i++) {
             logEntryWritesLatch = new CountDownLatch(2);
             logEntryWritesLatches.add(logEntryWritesLatch);
+            dataListeners.get(i).setSnapshotSync(false);
```

Comment:
Because we want to fence off the redundant async updates coming after the verfication of snapshot sync completes.  Once this flag is set, the listener will not process any updates where numEntries > 1.